scripts/apps/publish/directives/SubscribersDirective.ts

```diff
@@ -262,13 +264,15 @@ export function SubscribersDirective(
                 $scope.newDestination = null;
             };
 
-            $scope.$watchCollection('subscriber', (newValue, oldValue) => {
-                if (newValue && oldValue && newValue !== oldValue) {
+            $scope.$watch('subscriber', (newValue, oldValue) => {
+                if (newValue && oldValue && !_.isEqual(newValue, oldValue)) {
                     $scope.saveEnabled = true;
                 } else {
                     $scope.saveEnabled = false;
                 }
-            });
+            }, true);
```

Comment:
with deep watch I don't see a need for `isEqual` call, it should be always false